compiler/main.bal

```diff
@@ -13,9 +13,11 @@ public type Options record {
     boolean showTypes = false;
     int? debugLevel;
     // outDir also implies treating each file as a separate module
+    boolean redirectErrors = false;
```

Comment:
We don't need this option. Just send to stderr.  The point is just that a library module shouldn't decide where errors are sent.

Reply:
fixed